indico/htdocs/js/indico/Core/Dialogs/Users.js

```diff
@@ -1615,12 +1637,13 @@ type("UserListField", ["IWidget"], {
      */
     function(userDivStyle, userListStyle,
              initialUsers, includeFavourites, suggestedUsers,
-             allowSearch, enableGroups, conferenceId, privileges,
+             allowSearch, enableGroups, enableIpNetworks, conferenceId, privileges,
```

Comment:
I'd highly advice against adding new arguments anywhere but at the end for those huge constructors. Too easy to miss a case where things break